pkg/handler-launcher-com/cmd/v1/cmd.proto

```diff
@@ -95,6 +95,7 @@ message DiskInfo {
 message ClusterConfig{
   bool ExpandDisksEnabled = 1;
   bool FreePageReportingDisabled = 2;
+  bool BochsDisplayForEFIGuests = 3;
```

Comment:
We already have a mutating webhook for setting the VMI defaults: https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-api/webhooks/defaults.go

Wouldn't it be better to set the display for EFI guest there (taking into account the feature gate)? Thus, you will not need to touch handler-launcher comm.